Admittedly a build with cached intermediaries:

```
Step 15 : RUN npm install &&     rm -rf /root/.npm/
 ---> Running in e30afb08aad8
WARN engine ember-cli-app-version@1.0.0: wanted: {"node":">= 0.12.0"} (current: {"node":"0.10.36","npm":"3.5.0"})
npm ERR! Linux 4.2.6-200.fc22.x86_64
npm ERR! argv "node" "/usr/bin/npm" "install"
npm ERR! node v0.10.36
npm ERR! npm  v3.5.0
npm ERR! code E404

npm ERR! 404 Registry returned 404 for GET on http://registry.npmjs.org/roundcube-mail
npm ERR! 404 
npm ERR! 404  'roundcube-mail' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 It was specified as a dependency of 'roundcube-shell'
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! Please include the following file with any support request:
npm ERR!     /var/www/roundcube-shell.git/npm-debug.log
The command '/bin/sh -c npm install &&     rm -rf /root/.npm/' returned a non-zero code: 1
```

A fresh build from scratch though:

```
Step 15 : RUN npm install &&     rm -rf /root/.npm/
 ---> Running in 16042e274e3e
WARN engine ember-cli-app-version@1.0.0: wanted: {"node":">= 0.12.0"} (current: {"node":"0.10.36","npm":"3.5.0"})
npm ERR! Linux 4.2.6-200.fc22.x86_64
npm ERR! argv "node" "/usr/bin/npm" "install"
npm ERR! node v0.10.36
npm ERR! npm  v3.5.0
npm ERR! code E404

npm ERR! 404 Registry returned 404 for GET on http://registry.npmjs.org/roundcube-notifications
npm ERR! 404 
npm ERR! 404  'roundcube-notifications' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 It was specified as a dependency of 'roundcube-shell'
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! Please include the following file with any support request:
npm ERR!     /var/www/roundcube-shell.git/npm-debug.log
The command '/bin/sh -c npm install &&     rm -rf /root/.npm/' returned a non-zero code: 1
```

--- a/README.md
+++ b/README.md
@@ -18,16 +18,7 @@ You need to have a JMAP server running somewhere. Currently, the most mature ser
 
 ## Setup
 
-First, clone all the Roundcube apps you'll be using and run `npm link` inside each of their directories:
-
-* `git clone https://github.com/roundcube-next/roundcube-notifications` etc
-* `cd roundcube-notifications && npm link` etc
-
-Next, change into the `roundcube-shell` directory and run `npm link <app-name>` for each Roundcube app you'll be using:
-
-* `npm link roundcube-notifications` etc
-
-Third, bring in all dependencies and set up the project:
+Bring in all dependencies and set up the project:
 
 * `bundle`
 * `npm install`

--- a/package.json
+++ b/package.json
@@ -55,6 +55,8 @@
     "ember-sinon-qunit": "1.0.0",
     "ember-suave": "1.2.2",
     "ember-truth-helpers": "1.2.0",
-    "ember-try": "0.0.8"
+    "ember-try": "0.0.8",
+    "roundcube-notifications": "https://github.com/roundcube-next/roundcube-notifications.git",
+    "roundcube-mail": "https://github.com/roundcube-next/roundcube-mail.git"
   }
 }

